pytorch_lightning/plugins/training_type/sharded_spawn.py

```diff
@@ -114,12 +114,12 @@ def pre_backward(self, closure_loss: torch.Tensor) -> None:
     def post_training_step(self):
         pass
 
-    def new_process(self, trainer: "pl.Trainer") -> Optional["_SpawnOutput"]:
+    def pre_dispatch(self, trainer: "pl.Trainer") -> None:
```

Comment:
Is this override needed now? If not this could be cleanup for a separate PR

Reply:
I'm not sure yet, I need to investigate in #10988, because the ShardedScaler gets set at instantiation of the precision plugin which originally happens in the main process on Trainer init.